### PR TITLE
[EDGORDERS-55] - Logging Improvement

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose
https://issues.folio.org/browse/EDGORDERS-55

## Approach 
Logging has been improved for more productive troubleshooting.

### CODE REVIEW

File: src/main/java/org/folio/edge/orders/utils/OrdersOkapiClient.java [hunk 1]
```diff
@@ -41,15 +41,16 @@ protected void initDefaultHeaders() {
...
     final String method = routing.getProxyMethod() == null ? routing.getMethod() : routing.getProxyMethod();
 
     String resultPath = Optional.ofNullable(params)
       .map(it -> it.names().stream().reduce(routing.getProxyPath(), (acc, item) -> acc.replace(":" + item, params.get(item))))
       .orElse(routing.getProxyPath());
-
+    logger.info("send:: resultPath: {}", resultPath);
     switch (method) {
       case "POST":
+        logger.info("send:: Sending POST request to Okapi");
```
Comment (SerhiiNosko) [inline src/main/java/org/folio/edge/orders/utils/OrdersOkapiClient.java:53]: Can we add rounting object to such logs? Is it contains something usefull in toStrign implementation?

File: src/main/java/org/folio/edge/orders/OrdersHandler.java [hunk 2]
```diff
@@ -174,6 +179,7 @@ private void processErrorResponse(RoutingContext ctx, String respBody, String co
   }
 
   private void handleErrorResponse(RoutingContext ctx, int status, ResponseWrapper responseWrapper) {
```
Comment (SerhiiNosko) [inline src/main/java/org/folio/edge/orders/OrdersHandler.java:181]: I don't see that we log something before invoking handleErrorResponse and we do not logging anything in this method.
Lets add log here with warn level and log response wrapper into it(probably need to override toString for it)

File: src/main/resources/log4j.properties [hunk 3]
```diff
@@ -1,8 +1,15 @@
 # variables are substituted from filters-[production|development].properties
-log4j.rootLogger=INFO, CONSOLE
+#log4j.rootLogger=INFO, CONSOLE
```
Comment (SerhiiNosko) [inline src/main/resources/log4j.properties:2]: dont use commented lines

File: src/main/java/org/folio/edge/orders/utils/OrdersOkapiClient.java (hunk 1 above)
Comment (azizbekxm) [reply src/main/java/org/folio/edge/orders/utils/OrdersOkapiClient.java:53]: Ok, I will add

File: src/main/java/org/folio/edge/orders/OrdersHandler.java (hunk 2 above)
Comment (azizbekxm) [reply src/main/java/org/folio/edge/orders/OrdersHandler.java:181]: logger level will change and toString will be implemented for both ResponseWrapper and ErrorWrapper

**Review by SinghAdes — APPROVED** [review]

**Pull request merged**